app/views/shared/_footer.html.erb

```diff
@@ -1,6 +1,20 @@
 <footer class="bg-white dark:bg-gray-900" aria-labelledby="footer-heading">
   <h2 id="footer-heading" class="sr-only">Footer</h2>
-  <div class="mx-auto max-w-7xl px-6 pb-8 pt-16 sm:pt-24 lg:px-8">
+  <% unless current_page?(support_us_path) || current_page?(about_path) %>
+  <div class="mx-auto max-w-3xl px-8 pt-8 pb-8 xs:pt-24 lg:px-8">
+    <div class="flex flex-col sm:flex-row gap-8 :gap-4">
+      <div class="flex-1">
+        <h1 class="text-lg font-semibold text-gray-800 dark:text-gray-200">Help us shape the future of tech education!</h1>
+        <h2 class="text-sm text-gray-500 dark:text-gray-400">Join us in empowering learners around the globe by supporting The Odin Project!</h2>
+        <%= link_to 'Learn more about supporting us.', support_us_path, class: 'block prose-a text-blue-800 dark:text-blue-300 pt-4 font-semibold underline' %>
```

Comment:
One more (I promise), wouldn't it make more sense for this to use our general UI link styling (`text-gray-600 hover:text-gray-800 dark:text-gray-200 dark:hover:text-gray-300)? Right now it stands out a bit weird against the rest of say the home page, because its the only link to use these colors (or even blue in general).

Reply:
yep, i like that better.

thanks :D